### ISSUE TITLE
[BUG] Get-PnPWebPartXml changes web part properties

### ISSUE THREAD

**Issue by wonderplayer** [pnp/powershell#297]
### Reporting an Issue or Missing Feature
Executing Get-PnPWebPartXml on a list view web part enables "Display search box".
![image](https://user-images.githubusercontent.com/15668343/108223261-4be4d400-7142-11eb-9bdb-470593f4469c.png)

### Expected behavior 
Command gets XML and none of properties are changed.

### Actual behavior 
Command gets XML and enables "Display search box".

### Steps to reproduce behavior
1. Add list view web part to classic web part page
2. Get web parts from classic page with Get-PnPWebPart
3. Identify list view web part
4. Execute command Get-PnPWebPartXml on list web part

### What is the version of the Cmdlet module you are running?
1.3.5

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by wonderplayer** [comment]
I don't know where this bug really is.
The property that is responsible for in place search is InplaceSearchEnabled. You can get it by `Get-PnPWebPartProperty -ServerRelativePageUrl $pageUrl -Identity $webPart.id -Key "InplaceSearchEnabled"`.
When I get it for any list view web part I always get **true** no matter if the search box is visible or not. Even if I manually export web part I get **true** in XML.
But I can set it with script `Set-PnPWebPartProperty -ServerRelativePageUrl $pageUrl -Identity $webPart.id -Key "InplaceSearchEnabled" -Value $false` and the search box is really hidden. But when I try to get property after, it's set to **true** again and the search box isn't visible.
Could that be SharePoint Online bug? 🤔

**Comment by erwinvanhunen** [comment]
What happens here is that in order for us to be actually able to export the webpart we have to change the ExportMode of a webpart, as not all webparts are by default 'exportable'. We change it, save it, export it, and revert the value back to what it was. We're not setting any other property or change any other values. So this seems to be a behavior of the server that causes this.

**Comment by wonderplayer** [comment]
Yep, thought so that something strange was at play here.
By default, the search box is not visible, but the property is still **true**. Have 2 different Office 365 environments and they both have it like that.
So, what to do then?

**Comment by erwinvanhunen** [comment]
I debugged this case and it turns out that -regardless- of the setting made in the UI, the server reports it as true. Even before we factually export it.  So it's clearly a server issue. The only way forward is that you raise a support case with Microsoft about this. We can't do that for you.

I looked into a way of first retrieving all properties, then exporting the webpart and then restoring the original values. But if the initial retrieval of properties already contain incorrect values this has no use at all. For now the only way forward I can give you is to actually change the property to false again with Set-PnPWebPartProperty.
